pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -590,6 +590,9 @@ private static void enableCORS(final String origin, final String methods, final
             if (accessControlRequestMethod != null) {
                 response.header("Access-Control-Allow-Methods", accessControlRequestMethod);
             }
+            if (accessControlRequestMethod != null || accessControlRequestHeaders != null) {
+                response.header("Access-Control-Max-Age", "172800");
+            }
```

Comment:
This is an overdue change! Will prevent a separate OPTION call for before everything  single PATCH, GET, POST etc. There is max value that varies from browser to browser. Apparently for Firefox it maxes out at 24hours. Chromimum browsers max out in shorter periods. I went for 2 days here.